pages/setuppwa/setuppwa.tsx

```diff
@@ -0,0 +1,73 @@
+import { useRouter } from "next/router"
+import React from "react"
+
+const SetupPwa = () => {
+  const router = useRouter()
+  const [username, setUsername] = React.useState<string>("")
+
+  let accountUsername: string
+  const username_from_local = localStorage.getItem("username")
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+
+    if (accountUsername === "" || !accountUsername) {
+      localStorage.setItem("username", username)
+    }
+
+    router.push(
+      {
+        pathname: `/merchant/${username}`,
+      },
+      undefined,
+      { shallow: true },
+    )
+  }
+
+  if (username_from_local != null || username_from_local) {
```

Comment:
`username_from_local != null || username_from_local` can just be replaced with just `username_from_local`. Currently an empty string would result in true.